<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="creators_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>710</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IR Maker® Developped by Nathan Zwahlen, Benjamin Quiédeville &amp; Hugo Perrier</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="version_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>V1.1.0</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="boxes_layout">
     <item>
      <widget class="QGroupBox" name="sweep_box">
       <property name="title">
        <string>Sweep</string>
       </property>
       <widget class="QPushButton" name="browseSweep_button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse sweep file</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="begfreq_txtedit">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>55</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::LogicalMoveStyle</enum>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="endfreq_txtedit">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>80</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="begfreq_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>55</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Beginning freq.</string>
        </property>
       </widget>
       <widget class="QLabel" name="endfreq_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ending freq.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sweepgen_button">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>125</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Sweep generator</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="response_box">
       <property name="title">
        <string>Response</string>
       </property>
       <widget class="QPushButton" name="selectfolder_button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Select recording(s) folder</string>
        </property>
       </widget>
       <widget class="QListView" name="files_list">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>171</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="output_box">
       <property name="title">
        <string>Output</string>
       </property>
       <widget class="QCheckBox" name="mpt_checkbox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>MP transform</string>
        </property>
       </widget>
       <widget class="QComboBox" name="bitdepth_combo">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>150</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>16 bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24 bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32 bit</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="bitdepth_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Bit depth</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="autosave_radio">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>25</y>
          <width>21</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QRadioButton" name="customsave_radio">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Custom output</string>
        </property>
       </widget>
       <widget class="QPushButton" name="browseout_button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse output</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="autosave_label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Automatic output (create folder &amp; autonaming)</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>701</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="PlotWidget" name="temporal_plot"/>
     </item>
     <item>
      <widget class="PlotWidget" name="spectral_plot"/>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="compute_button">
     <property name="text">
      <string>Create IR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="play_ir">
     <property name="text">
      <string>Play IR</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
